### PR TITLE
v3.11.0 locale support added for string

### PR DESCRIPTION
**Pull request by ishaileshmishra**
Hi @uttamukkoji , Please review the code.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
-    }
+apply plugin: 'io.github.gradle-nexus.publish-plugin'
+apply from: "${rootDir}/scripts/publish-root.gradle"
+
+//allprojects {
+//    repositories {
+//        mavenCentral()
+//        google()
+//    }
+//}
```
Comment (uttamukkoji) [inline build.gradle:22]: @ishaileshmishra remove commented code

Comment (ishaileshmishra) [reply build.gradle]: Removed, thank you

**Review by uttamukkoji — APPROVED** [review]
Approved code review. @ishaileshmishra Please check all Actions are passing.

**Comment by ishaileshmishra** [comment]
Awaiting the SRE team to grant access to the specified **Setting** and to include the confidential **local properties** credentials in the secure section, as they cannot be committed publicly for security reasons.

**Review by aravindbuilt — APPROVED** [review]

**Pull request merged**